corehq/apps/userreports/reports/filters.py

```diff
@@ -52,7 +52,9 @@ def __init__(self, filter, value):
     def to_sql_filter(self):
         if self.value is None:
             return ""
-        return "{0} {1} :operand".format(self.filter.field, self.value['operator'])
+        ret = BasicFilter(self.filter.field, 'operand')
+        ret.operator = self.value['operator']
```

Comment:
you also have to bump the version in setup.py

https://confluence.dimagi.com/display/commcarehq/Python+Packaging+Crash+Course